config/config.go

```diff
@@ -892,6 +892,7 @@ func SetupViper(v *viper.Viper, filename string) {
 	v.SetDefault("adapters.madvertise.endpoint", "https://mobile.mng-ads.com/bidrequest{{.ZoneID}}")
 	v.SetDefault("adapters.marsmedia.endpoint", "https://bid306.rtbsrv.com/bidder/?bid=f3xtet")
 	v.SetDefault("adapters.mediafuse.endpoint", "http://ghb.hbmp.mediafuse.com/pbs/ortb")
+	v.SetDefault("adapters.medianet.endpoint", "https://prebid-adapter.media.net/rtb/pb/prebids2s?src={{.Host}}")
```

Comment:
I don't see any logic in adapter code to handle `{{.Host}}` placeholder. 
Please refer to this example: adkernel.go -> func `buildEndpointURL`

Reply:
Removed, no need for it